### ISSUE TITLE
Discourage use of disclosure (detail / summary) widgetsy

### ISSUE THREAD

**Issue by juleskuehn** [wet-boew/wet-boew#8874]
A lot of GC sites, instead of using a table of contents, hide the majority of their content inside disclosure widgets. For example, see the [Guideline on Making IT Usable by All](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32620). This example has an "Expand all" button thankfully, but not all do.

Hiding content in these widgets make the site harder to "skim", and will ultimately reduce the amount of information that page visitors take in. And while WET makes these widgets fairly accessible, they are much less accessible to adaptive technology than simply not having them. (For instance, subheadings, links etc. cannot be seen until expanded). AT support for expand/collapse voicing is imperfect as well.

A good first principle of interaction design is to minimize the required interaction.
In fact, “[interaction cost](https://www.nngroup.com/articles/interaction-cost-definition/)” (the amount of work required from the user to find the information they need) is a core heuristic for usability.

If pages are long, and headings of the page need to be presented sooner, a table of contents would be more appropriate.

**I suggest we add guidance discouraging the use of these widgets for primary page content.**

**Comment by thekodester** [comment]
What do you think of detail/summary being used in combo with table of summary?

This is not a WET project, but an idea of what I mean: https://thekodester.ca/legal

There also an example of this on wet with https://wet-boew.github.io/v4.0-ci/demos/toggle/toggle-en.html and https://wet-boew.github.io/v4.0-ci/theme/content-secmenu-en.html

Just some ideas.

**Comment by juleskuehn** [comment]
No... content should be visible on page ready by default. I can dig up further resources to support this position if necessary, but to me its pretty obvious.
